components/CreateAskForm.tsx

```diff
@@ -1,5 +1,5 @@
 import { useState } from "react";
-import styles from "@/styles/components/create-ask-form.module.scss";
+import styles from "../styles/components/create-ask-form.module.scss";
```

Comment:
It didn't let you use `@` because you forgot to define the custom paths in tsconfig

Reply:
noted, still didnt work for some...